src/Sentry.AzureFunctions.Worker/Sentry.AzureFunctions.Worker.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+      <TargetFrameworks>net7.0;net6.0</TargetFrameworks>
```

Comment:
Unless there's something special we need that's only open to .NET 7, then please target .NET 6 only in the source libraries.  Thanks.